src/main/resources/templates/createAccount.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org" lang="en">
+
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,100..700;1,100..700&display=swap" rel="stylesheet">
+    <title>Frontpage</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../public/css/styles.css" th:href="@{/css/styles.css}" rel="stylesheet"/>
+</head>
+
+<body>
+
+<img class="headerImage" src="../public/images/CupcakeLogo.png" th:src="@{/images/CupcakeLogo.png}"/>
+
+<form class="generic-form" method="post">
+    <input type="text" name="userEmail" placeholder="Indtast E-mail"/>
+    <input type="password" name="password1" placeholder="Indtast kodeord"/>
+    <input type="password" name="password2" placeholder="Gentag kodeord"/>
+    <br>
+
+    <button class="buttons" type="submit" formaction="/createAccount">Opret konto</button>
+    <div class="fontSizeForMessages"><p th:text="${message}"></p></div>
+
+</form>
+
+<div>
+    <a th:if="${session.currentUser != null}" href="index.html" th:href="@{/logout}">Logout</a>
+</div>
```

Comment:
Remove unnecessary logout. The user should not be allowed to access the create user page if they are already logged in so this is not necessary. Instead a check would be added to the UserController.createUser routing, checking if a user is already logged in (sessionAttribute) and if so, load the user-frontpage.html